pkg/distro/fedora/package_sets.go

```diff
@@ -177,17 +177,65 @@ func iotCommitPackageSet(t *imageType) rpmmd.PackageSet {
 		},
 	}
 
+	switch t.arch.Name() {
```

Comment:
For a follow-up: this kind of logic would be perfect to have in the Platform itself.